src/components/datagrid/data_grid_types.ts

```diff
@@ -846,6 +882,14 @@ export interface EuiDataGridToolBarVisibilityDisplaySelectorOptions {
    * When `false`, removes the ability to change row height display through the UI
    */
   allowRowHeight?: boolean;
+  /**
+   * When `false`, removes the ability to reset styles to default through the UI
+   */
+  allowResetButton?: boolean;
+  /**
+   * If passed a React node it will append the additional option to the bottom of the display settings popover
+   */
+  additionalDisplaySettings?: ReactNode;
```

Comment:
Nice job with these prop names and descriptions! Some minor copy suggestions:

```suggestion
  allowResetButton?: boolean;
  /**
   * Allows appending additional content to the bottom of the display settings popover
   */
  additionalDisplaySettings?: ReactNode;
```